CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased](https://github.com/jenkinsci/analysis-model/compare/analysis-model-4.0.0...master)
 
+### Changed
+- \[API\]: `YuiCompressorParser` uses new base class 
+[LookaheadParser](https://github.com/jenkinsci/analysis-model/blob/master/src/main/java/edu/hm/hafner/analysis/LookaheadParser.java) 
+that provides a lookahead of the next report line instead of using multi line parsing.
+
```

Comment:
Since 5.0.0 has not been released yet: just add your changed item to the existing `### Changed` block. I will add the 5.0.0 section right after the release.